Frontend/CO.CDP.OrganisationApp/Pages/Registration/JoinOrganisation.cshtml

```diff
@@ -6,7 +6,14 @@
     var organisationDetails = Model.OrganisationDetails!;
     var userDetails = Model.UserDetails;
     var joinHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Join)).HasInnerHtml;
-    ViewData["Title"] = $"{@StaticTextResource.OrganisationRegistration_JoinOrganisation_Heading}: {@organisationDetails.Name}";
+    ViewData["Title"] = string.Format(
+        StaticTextResource.OrganisationRegistration_JoinOrganisation_Heading,
+        organisationDetails.Name
+    );
+    ViewData["RadioQuestionTitle"] = string.Format(
```

Comment:
I would just do this inline below, on line 81 - the `ViewData` stuff is only done because we need that in order to pass the `Title` back up to the layout template.

Reply:
Moved